code/modules/mob/living/carbon/human/species/monkey.dm

```diff
@@ -46,6 +46,10 @@
 /datum/species/monkey/get_random_name()
 	return "[lowertext(name)] ([rand(100,999)])"
 
+/datum/species/monkey/handle_death(gibbed, mob/living/carbon/human/H)
+	. = ..()
+	SSmobs.cubemonkeys -= H
```

Comment:
Issue with handling it on the species is that if they're humanized before they're killed it won't delete them from the list properly. Don't know how to work around this unless we change the expand proc to use a monkey subtype and then overwrite it's destroy proc like TG did

Reply:
monkey faction?

Reply:
that'd be pretty snowflake since it's not really the intended use of factions, could have it removing from the carbon/human/death proc but that wouldn't be optimal too because the code would run for every non-monkey as well

Reply:
You could safely put this in `human/death`

It's a needless check 99% of the time, but it will work.